sample/golang-echo/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.9-alpine
```

Comment:
I meant LTS of Golang. By the way - where are you seeing the bad flags on the images? How about 1.8-alpine3.6?